app/views/layouts/themes/default/_header.html.haml

```diff
@@ -8,8 +8,5 @@
         .city
           = current_website.city
     %nav#main-nav
-      - current_website.navigation_page_names_and_slugs.each do |page_name, page_slug|
-        = link_to page_name, page_path(website_event_slug, page_slug)
-      = link_to "Schedule", schedule_path(current_website.event)
-      = link_to "Program", program_path(website_event_slug)
-      = link_to "Sponsors", sponsors_path(website_event_slug)
+      - current_website.navigation_links.each do |slug|
+        = link_to current_website.link_options.key(slug), page_path(website_event_slug, slug)
```

Comment:
This is a bit sneaky. Since the `/schedule`, `/program` and `/sponsors` routes are structured just like a page route we can steal this page_path helper and treat the link slugs all the same.